modules/core/keeper/msg_server_test.go

```diff
@@ -827,7 +831,86 @@ func (suite *KeeperTestSuite) TestUpdateClientParams() {
 }
 
 // TestIBCSoftwareUpgrade tests the IBCSoftwareUpgrade rpc handler
-func (*KeeperTestSuite) TestIBCSoftwareUpgrade() {
+func (suite *KeeperTestSuite) TestIBCSoftwareUpgrade() {
+	var (
+		expError error
```

Comment:
super nit: I like defining the expError on the testCases, it makes it more explicit what we expect to occur and avoids potential mishaps of not setting an expected error